### PR TITLE
PHPC-1159: Test on specific server versions in Travis CI

### PR DESCRIPTION
**Pull request by kvwalker**
https://jira.mongodb.org/browse/PHPC-1159

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,33 +1,53 @@
...
     - php: 7.2
-
-services:
-  - mongodb
-
-env:
-  global:
-    - TEST_PHP_ARGS="-q -s output.txt -g XFAIL,FAIL,BORK,WARN,LEAK,SKIP -x --show-diff"
-    - REPORT_EXIT_STATUS=1
+    - php: 7.0
```
[the remark this answers is not in the document]
Comment (jmikola) [reply .travis.yml]: No reason in particular. The test for PHPLIB just happened to use PHP 7.0 to test older server versions.

File: .travis.yml [hunk 2]
```diff
@@ -1,33 +1,53 @@
...
+        - SERVER_VERSION=3.0.15
+    - php: 7.0
+      env:
+        - SERVER_VERSION=3.2.18
+    - php: 7.0
+      env:
+        - SERVER_VERSION=3.4.11
 
 before_install:
+  - pip install "mongo-orchestration>=0.6.7,<1.0" --user `whoami`
```
[the remark this answers is not in the document]
Comment (jmikola) [reply .travis.yml]: I expect this is due to older version of Python on the Travis images. Perhaps @ShaneHarvey can chime in here.

File: tests/cursor/cursor-tailable-003.phpt [hunk 3]
```diff
@@ -34,12 +34,12 @@ $startTime = microtime(true);
 echo "Awaiting results...\n";
 $it->next();
 printf("Waited for %.6f seconds\n", microtime(true) - $startTime);
-
+// Sometimes the cursor will wait for 0.09999 seconds and sometimes it will wait for 0.1000.
```
Comment (jmikola) [inline tests/cursor/cursor-tailable-003.phpt]: Did you mean `0.0099` and `0.01`, which is 9.9ms and 10ms respectively? As-is, this comment doesn't agree with the pattern below, as you're always expecting `0.0` and that would conflict with a possible `0.1000`.

Comment (kvwalker) [reply tests/cursor/cursor-tailable-003.phpt]: Yes, that's definitely what I meant, thanks!

File: .travis.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jmikola) [reply .travis.yml]: Per @behackett, we don't need to worry about this warning. A legitimate concern would be if the version of OpenSSL on these old CI environments causes connectivity issues due to not supporting TLS v1.2, but that's a completely different class of error.

**Comment by jmikola** [comment]
This may need another rebase due to recent changes in `tests/manager/manager-invalidnamespace.phpt`. I believe I split up the `SKIPIF` functions each to their own line.

Beyond that, I'm LGTM but we can wait until @derickr acknowledges the comments above.

**Pull request merged**